contracts/actions/flashloan/FLEuler.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+pragma experimental ABIEncoderV2;
```

Comment:
Can be removed, I see we have it in a few places but no need in >0.8 versions is using ABIEncoderV2 as default